client/me/form-base/index.js

```diff
@@ -39,16 +40,21 @@ export default {
 		return {
 			redirect: false,
 			submittingForm: false,
+			formsSubmitting: {},
 			changingUsername: false,
 			usernameAction: 'new',
 			showNotice: false,
 		};
 	},
 
-	showNotice: function () {
+	showNotice: function ( formName ) {
 		if ( this.props.userSettings.initialized && this.state.showNotice ) {
 			notices.clearNotices( 'notices' );
-			notices.success( this.props.translate( 'Settings saved successfully!' ) );
+			notices.success(
```

Comment:
Hey folks! We've been reduxifying notices in #48408 and the ones in this file are among the few left to address.

Would you like to refactor them to use Redux instead? If you're interested, I'd love to help with anything you need.

Reply:
👋 @tyxla Thank you. We've just discussed this with @flootr over at p1612195802434000-slack-ajax. We're going to get this merged and then Redux can land if that's ok?

Reply:
Sounds good @getdave! Thanks for confirming 👍